.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: Test
```

Comment:
Perhaps we could rename this and the filename to reflect that it both tests and possibly deploys?

Reply:
```suggestion
name: Test & Deploy
```